wiki/People/The_Team/zh.md

```diff
@@ -1,56 +1,87 @@
----
-outdated: true
----
-
 # The Team
 
+以下列出的玩家是 osu!team 的核心成员，并且致力于让这个游戏变得更有趣。
+
 ## 游戏创建者
 
-| 名 | 描述 |
-| ---- | ----------- |
-| [peppy](https://osu.ppy.sh/users/2) | 项目领头人 |
+| 昵称                                             | 介绍       |
```

Comment:
the current table header seems more accurate to the English version, but if this is better then also change the header on line 13